psychopy/tests/test_misc/test_event.py

```diff
@@ -226,6 +227,20 @@ def setup_class(self):
         self.win = Window([128,128], monitor=mon, winType='pyglet', pos=[50,50], autoLog=False)
         assert pygame.display.get_init() == 0
 
+    def test_shutdown_key(self):
+        key = 'escape'
```

Comment:
It's not a mystery at all:
```python
In [4]: keys = 'escape'

In [5]: 'escape' in keys
Out[5]: True
```
But we do get side-effects:
```python
In [6]: 'e' in keys
Out[6]: True
```

Now I wonder – do we _really_ need support for a _list_ of keys? Or wouldn't a single shutdown key suffice? Then we wouldn't need special handling / have to force users to provide even single keys as a list.